### ISSUE TITLE
Couldn't find WMP

### ISSUE THREAD

**Issue by ZachjuKamashi** [T0biasCZe/Windows-Media-Player-Discord-RPC#4]
As the title says, it couldn't find WMP which is weird, I have it open, I've tried running the application as an administrator and it just stays stuck at that error? Also when I maximize the "form" window, you can see the built in WMP which is probably used to detect what music is playing, not sure if you are meant to see it or not.

**Comment by ZachjuKamashi** [comment]
Found the problem and fixed it

**Comment by T0biasCZe** [comment]
> Found the problem and fixed it

What was the issue? Incase anyone in the future has the same issue

**Comment by ZachjuKamashi** [comment]
> > Found the problem and fixed it
> 
> What was the issue? Incase anyone in the future has the same issue

I believe I was using the custom app ID provided if you do not want to show album covers "1076519967631093891" and for some reason because of this windows media player couldn't be detected. I switched over to the application ID through discord developers and that worked fine. But TBH even then I noticed several other bugs.

It was showing the image wmp_empty.png on discord despite it also saying a song was playing in which case it should have shown wmp_icon.png if I understood the wording correct on the images chosen. 

There is also the elapsed section. I'm pretty sure when I saw the VLC discord rich present function a few years ago it showed how far you were into the song but here it continues infinitely even if paused which is very odd. Actually wait a minute, after retesting it as of writing now does kinda work as intended? Well this is very finicky then, it either works or doesn't in some occasions. As for reasons why, I have no idea.

On another note the image sometimes refuses to appear at all. Leaving it blank and kinda look a little weird.

![Screenshot (455)](https://github.com/T0biasCZe/Windows-Media-Player-Discord-RPC/assets/98941240/5fcc48a6-0953-453f-b587-8311e0e9b91c)
